cdap-docs/faqs/source/general.rst

```diff
@@ -0,0 +1,84 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Frequently Asked Questions about the Cask Data Application Platform
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+:titles-only-toc: true
+
+.. _faqs-general-questions:
+
+=======================
+FAQs: General Questions
+=======================
+
+What is available in the CDAP SDK?
+----------------------------------
+The CDAP SDK includes:
+
+- Local instance of the CDAP Server;
+- CDAP APIs for building applications;
+- CDAP UI and CDAP CLI for interacting with CDAP; and
+- Examples for getting started with CDAP.
+
+
+What platforms are supported by the Cask Data Application Platform SDK?
+-----------------------------------------------------------------------
+The CDAP SDK has been extensively tested on Mac OS X and Linux. CDAP on Windows has not
+been extensively tested. If you have any issues with CDAP on Windows, help us by 
+`filling a ticket <https://issues.cask.co/browse/CDAP>`__.
+
+What programming languages are supported by CDAP?
+-------------------------------------------------
+CDAP currently supports Java for developing applications.
+
+What version of Java SDK is required by CDAP?
+---------------------------------------------
+The latest version of the JDK or JRE version 7 or version 8 must be installed
+in your environment; we recommend the Oracle JDK.
+
+What version of Node.js is required by CDAP?
+--------------------------------------------
+We recommend any version of `Node.js <https://nodejs.org/>`__ |node-js-version|.
+
+I have a Hadoop cluster in my data center, can I run CDAP that uses my Hadoop cluster?
+--------------------------------------------------------------------------------------
+Yes. You can install CDAP on your Hadoop cluster. See the :ref:`Installation procedure <install>`.
```

Comment:
Change answer to "Yes. You can install CDAP distributed on your Hadoop cluster. See Installation procedure."

You're missing the word 'distributed'.